src/components/Item/VisualizationItem/ItemHeaderButtons.js

```diff
@@ -78,24 +78,25 @@ const ItemHeaderButtons = props => {
 
     const ViewAsMenuItems = () => (
         <>
-            <MenuItem
-                dense
-                active={activeType === CHART}
-                label={i18n.t('View as Chart')}
-                onClick={onViewChart}
-                icon={<ChartIcon style={iconFill} />}
-            />
-            <MenuItem
-                dense
-                active={[REPORT_TABLE, EVENT_REPORT].includes(activeType)}
-                label={i18n.t('View as Table')}
-                onClick={onViewTable}
-                icon={<TableIcon style={iconFill} />}
-            />
-            {itemHasMapView() && (
+            {activeType !== CHART && (
```

Comment:
Should this include `EVENT_CHART` as well? @janhenrikoverland @jenniferarnesen

Reply:
Yes

Reply:
Fixed it.

Reply:
Thanks!